homeassistant/components/tuneblade_remote/manifest.json

```diff
@@ -0,0 +1,21 @@
+{
+  "domain": "tuneblade_remote",
+  "name": "TuneBlade Remote",
+  "codeowners": ["@spycle"],
+  "config_flow": true,
+  "dependencies": ["media_player", "http"],
+  "dhcp": [],
+  "documentation": "https://www.home-assistant.io/integrations/tuneblade_remote",
+  "integration_type": "hub",
+  "iot_class": "local_polling",
+  "mqtt": [],
+  "quality_scale": "bronze",
+  "requirements": ["pytuneblade==0.1.1"],
+  "usb": [],
+  "zeroconf": [
+    {
+      "type": "_http._tcp.local.",
+      "name": "*une?lade._http._tcp.local."
```

Comment:
why the *une? ? Doesn't that mean that the `e` is optional?

Reply:
Oh right, I would rather say do `[Tt]`, I don't believe * at the start is valid

Reply:
but it has to be * at the start. The actual broadcast is <your-pc-name-here>@TuneBlade._http._tcp.local.

I've gone with "*[Tt]une[Bb]lade._http._tcp.local." but need to test it

Reply:
Yes, but `*` means that it will do zero or more of the thing in front of it, and there is nothing in front of it, so this wouldn't be valid regex